src/management/cameras.php

```diff
@@ -0,0 +1,109 @@
+<?php
+$servername = "db";
+$username = "db_username";
+$password = "db_password";
+$dbname = "db_name";
+
+// Create connection
+$conn = new mysqli($servername, $username, $password, $dbname);
+
+// Check connection
+if ($conn->connect_error) {
+  die("Connection failed: " . $conn->connect_error);
+}
+
+// Fetch city name from the GET parameters
+$city = $_GET['city'];
+
+// Fetch cameras for the selected city
+$camerasQuery = "SELECT * FROM cameras
+                 INNER JOIN parking_spaces ON cameras.parking_space_id = parking_spaces.id
+                 WHERE parking_space.city = '$city'";
+$camerasResult = $conn->query($camerasQuery);
```

Comment:
There's a typo in WHERE clause - `parking_space`.
Also, code contains SQL Injection security issue, `$city` is not escaped. I recommend to use [prepared statements](https://www.php.net/manual/en/mysqli.quickstart.prepared-statements.php)

```suggestion
$stmt = $conn->prepare("SELECT * FROM cameras
                 INNER JOIN parking_spaces ON cameras.parking_space_id = parking_spaces.id
                 WHERE parking_spaces.city = ?");
$stmt->bind_param('s', $city);
$stmt->execute();
$camerasResult = $stmt->get_result();
```